src/main/scala/cromwell/engine/callactor/FinalCallActor.scala

```diff
@@ -0,0 +1,14 @@
+package cromwell.engine.callactor
+
+import akka.actor.ActorRef
+import cromwell.engine.WorkflowDescriptor
+import cromwell.engine.callexecution.CallExecutionActor
+import cromwell.engine.workflow.FinalCallKey
+
+class FinalCallActor(override val key: FinalCallKey) extends CallActor {
```

Comment:
I'm assuming there's an obvious reason this can't be a case class and I'm just too daft to spot it :)